index.html

```diff
@@ -14,41 +15,139 @@
     />
 
   -->
-    <link rel="stylesheet" href="style.css" />
-
-    <script
-      src="https://kit.fontawesome.com/26c38390b4.js"
-      crossorigin="anonymous"
-    ></script>
-  </head>
-  <body>
-    <header class="navbar">
-      <a href="#" class="brand-title">My logo</a>
-      <i class="fas fa-bars"></i>
-    </header>
+  <link rel="stylesheet" href="style.css" />
+
+  <script src="https://kit.fontawesome.com/26c38390b4.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+
+  <header class="navbar">
+    <a href="#" class="brand-title">My logo</a>
+    <i class="fas fa-bars"></i>
+  </header>
+
+  <container class="page">
 
     <section class="presentation">
-      <h2>I’m Alexandra. Glad to see you!</h2>
+      <h2>I’m Gonzalo. Glad to see you!</h2>
+
+      <article class="job">
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+      </article>
+      <section class="contact">
+        <p>LET’S CONNECT</p>
+      </section>
+      <section class="socialmedia">
+        <ul>
+          <li><i class="fa-brands fa-twitter"></i></li>
+          <li><i class="fa-brands fa-linkedin"></i></li>
+          <li><i class="fa-solid fa-m"></i></li>
+          <li><i class="fa-brands fa-square-github"></i></li>
+          <li><i class="fa-regular fa-hand-peace"></i></li>
+        </ul>
```

Comment:
- [ ] These social media icons should stick to the Figma design , it means you should make sure they have the same style as the one found in the Figma design. Kindly improve that 👍 
![Capture d’écran 2023-03-29 à 2 46 21 AM](https://user-images.githubusercontent.com/80458199/228400915-40194fe2-9473-4765-b1ce-29f83c29bf17.png)